src/systemd/sd-bus.h

```diff
@@ -94,15 +94,15 @@ enum {
         _SD_BUS_CREDS_ALL               = (1ULL << 34) -1
 };
 
-enum {
+__extension__ enum {
         SD_BUS_NAME_REPLACE_EXISTING  = 1ULL << 0,
         SD_BUS_NAME_ALLOW_REPLACEMENT = 1ULL << 1,
         SD_BUS_NAME_QUEUE             = 1ULL << 2
 };
 
-enum {
+__extension__ enum {
         SD_BUS_MESSAGE_DUMP_WITH_HEADER  = 1ULL << 0,
-        SD_BUS_MESSAGE_DUMP_SUBTREE_ONLY = 1ULL << 1,
```

Comment:
We usually append a comma also in the last entry, to make diffs smaller when we extend enums. Why do you want to drop them?

Reply:
These are not allowed with `-ansi`, which the public headers are supposed to target.